packages/bs-ethereum/src/BlockscoutNeoXBDSEthereum.ts

```diff
@@ -336,17 +376,25 @@ export class BlockscoutNeoXBDSEthereum extends RpcBDSEthereum {
     }
 
     erc20Balances.forEach(balance => {
-      const token: Token = {
-        decimals: parseInt(balance.token.decimals),
-        hash: balance.token.address,
-        name: balance.token.symbol,
-        symbol: balance.token.symbol,
-      }
+      try {
+        if (balance.token.type !== 'ERC-20') {
+          return
+        }
 
-      balances.push({
-        amount: ethers.utils.formatUnits(balance.value, token.decimals),
-        token,
-      })
+        const token: Token = {
+          decimals: balance.token.decimals ? parseInt(balance.token.decimals) : 18,
```

Comment:
Wouldn't it be nice if you put "18" in an explainable variable like `DEFAULT_DECIMALS`?